src/StaticCaching/DefaultInvalidator.php

```diff
@@ -51,6 +52,20 @@ protected function invalidateTermUrls($term)
     {
         if ($url = $term->url()) {
             $this->cacher->invalidateUrl($url);
+
+            $term->taxonomy()->collections()->each(function ($collection) use ($term) {
+                if (! ($url = $collection->url($term->site()->handle()))) {
+                    return;
+                }
+
+                $url = vsprintf('%s%s/%s', [
+                    Str::ensureRight($url, '/'),
+                    $term->taxonomy()->handle(),
+                    $term->slug(),
+                ]);
```

Comment:
Instead of all this, you can do `$term->collection($collection)->url()`

Reply:
Ah didn’t know you could get to the collection like that. You’d still have to concatenate the taxonomy handle and term slug parts though, right?

Reply:
Oh haha 😆 so simple. Thanks. There’s benefit in knowing the API 🙃
Will make the change.